src/wazuh_db/wdb.c

```diff
@@ -146,6 +146,7 @@ static const char *SQL_STMT[] = {
     [WDB_STMT_GLOBAL_GET_AGENTS_BY_LESS_KEEPALIVE] = "SELECT id FROM agent WHERE id > ? AND last_keepalive < ? LIMIT 1;",
     [WDB_STMT_GLOBAL_GET_AGENTS_BY_CONNECTION_STATUS] = "SELECT id FROM agent WHERE id > 0 AND connection_status = ?;",
     [WDB_STMT_GLOBAL_GET_AGENT_INFO] = "SELECT * FROM agent WHERE id = ?;",
+    [WDB_STMT_GLOBAL_GET_AGENTS_TO_DISCONNECT] = "SELECT id FROM agent WHERE id > 0 AND connection_status = 'active' AND last_keepalive < ?;",
```

Comment:
Shouldn't we take 'pending' agents too in this query?

Reply:
I wouldn't take them. I think that it would be useful for the user to know that an agent for some reason stayed in `pending` state. Is true that by turning the manager off and on, the agent will be set as `disconnected`, but I think that is a different scenario. If the manager was never turned off, I would like to know whether an agent was not able to achieve the `active` state and stayed indefinitely in `pending` state.

Reply:
Ok. Sounds logic. It will avoid triggering "disconnection" alerts when the agent not even connected.